packages/fluent-editor/src/config/types/editor-config.interface.ts

```
@@ -14,10 +14,13 @@ export interface IEditorConfig extends QuillOptions {
   autoProtocol?: boolean | string
   editorPaste?: any
   uploadOption?: {
-    imageUploadToServer?: boolean
+    imageUpload?: ({ file: File, callback, editor}) => void
     imageAccept?: Array<string>[] | string
     fileAccept?: Array<string>[] | string
     isVideoPlay?: boolean
+    maxSize?: Number
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)